apisix/plugins/multi-auth.lua

```diff
@@ -54,6 +54,10 @@ function _M.check_schema(conf)
                 if auth.type ~= 'auth' then
                     return false, auth_plugin_name .. " plugin is not supported"
                 end
+                local ok, err = auth.check_schema(auth_plugin_conf, auth.schema)
+                if not ok then
+                    return false, "plugin " .. auth_plugin_name .. " check schema failed: " .. err
```

Comment:
can you also add a test case which shows this error would be raised if any auth_plugin has wrong configuration.

Reply:
ok

Reply:
perfect, really appreciate it <3. I will re-review once the CI has completed running.